packages/core/scripts/collateral-umip/3_Verify.js

```diff
@@ -25,22 +31,38 @@ async function runExport() {
 
   const collaterals = _.castArray(argv.collateral);
   const fees = _.castArray(argv.fee);
-  const decimals = argv.decimals && _.castArray(argv.decimals);
+  const decimals = _.castArray(argv.decimals);
+  const polygonCollaterals = _.castArray(argv.polygonCollateral);
 
   const argObjects = _.zipWith(collaterals, fees, decimals, (collateral, fee, numDecimalsArg) => {
     return { collateral, fee, numDecimalsArg };
   });
 
+  const whitelist = await AddressWhitelist.deployed();
+
   for (const { collateral, fee, numDecimalsArg } of argObjects) {
     const decimal = await getDecimals(collateral, numDecimalsArg, ERC20);
 
     const store = await Store.deployed();
     assert.equal((await store.computeFinalFee(collateral)).rawValue, parseUnits(fee, decimal).toString());
 
-    const whitelist = await AddressWhitelist.deployed();
     assert(await whitelist.isOnWhitelist(collateral));
   }
 
+  // Check for latest event RelayedGovernanceRequest event emitted by GovernorRootTunnel. We can't query for more events
```

Comment:
For some reason I can't grab more than one event for this contract and I suspect that its due to the connection to a ganache fork.